src/fields/Image.jsx

```diff
@@ -1,13 +1,14 @@
-var React = require("react");
-var cleanProps = require("./clean-props");
-var MultiplicityField = require('./MultiplicityField.jsx');
+import React from 'react';
```

Comment:
Not sure why but I got this error when I tried to make the above change.

<img width="392" alt="screen shot 2017-07-30 at 11 26 29 pm" src="https://user-images.githubusercontent.com/2896608/28765402-6cf95b42-757f-11e7-8a53-18e526d5cb64.png">